Winium/Winium.StoreApps.Driver/EmulatorHelpers/Deployer.cs

```diff
@@ -155,6 +155,25 @@ public void SendFiles(Dictionary<string, string> files)
             }
         }
 
+        public void InstallDependencies(List<string> dependencies )
+        {
+            if (dependencies == null || !dependencies.Any())
+            {
+                return;
+            }
+
+            foreach (var dependency in dependencies)
+            {
+                InstallDependency(dependency);
+            }
+        }
+
+        public void InstallDependency(string path)
+        {
+            var appManifest = Utils.ReadAppManifestInfoFromPackage(path);
+            Utils.InstallApplication(this.deviceInfo, appManifest, DeploymentOptions.None, path);
+        }
```

Comment:
Seems I was a bit too quick, it's needed by `RemoteApplication` too ( it reads `ProductId` from it. ) So a better way would probably be to have `InstallApplicationPackage` take the `AppManifest` then let `Install` create the `AppManifestInfo` and set `RemoteApplication`